### ISSUE TITLE
Can't do correct upsampling

### ISSUE THREAD

**Issue by Alexasandrovich** [mahmoudnafifi/HistoGAN#15]
Hi! Unable to do correct upsampling in rehistoGAN.py on my dataset. That is, if you take the standard 256x256 network output, the colours change normally. But if you add --upsampling_output option, the picture becomes faded, i.e. it gets predominantly white. I changed the dimensionality to be a multiple of two, but it didn't help either. 

Running it like this:
python rehistoGAN.py --name Universal_rehistoGAN_v0 --generate True --input_image my_inp_img.png --target_hist my_goal_img.png --gpu 0 --network_capacity 18 -upsampling_output True

What could be the problem? Thanks in advance for the answer

P.S. The pictures show a person, some buildings, trees and bushes, i.e. the image was taken inside the city

**Comment by mahmoudnafifi** [comment]
Could you please post the image before and after to have a better understanding of the issue you’ve faced?

**Comment by Alexasandrovich** [comment]
> Could you please post the image before and after to have a better understanding of the issue you’ve faced? 

https://drive.google.com/folderview?id=1vIvQW93ddpLwnYeRMDKKsAJYxl3iewFG

Generated pictures with --upsampling_output (predominantly white color) and without

**Comment by mahmoudnafifi** [comment]
Interesting! Probably the bug is that the reference image in the upsampling is in 0-255 range? Since you have the setup of HistoGAN in your device, do you mind checking the value of reference image after this [line](https://github.com/mahmoudnafifi/HistoGAN/blob/50be1482638ace3ec85d733e849dec494ede155b/ReHistoGAN/rehistoGAN.py#L1145)

**Comment by mahmoudnafifi** [comment]
Please post here if you find the reason of the issue. Thanks!

**Comment by Alexasandrovich** [comment]
> Please post here if you find the reason of the issue. Thanks!

In method pyramid_upsampling variable "target" is in range [0, 255], but reference is in range [0, 1]. 
I fix it as "target = torch.squeeze(target, dim=0).cpu().detach().numpy() / 255" in the same method

**Comment by mahmoudnafifi** [comment]
Do you mind creating a PR with this correction?
